proceedures.md

```diff
@@ -0,0 +1,100 @@
+TODO:
++ communicating with hosts
++ finding a host
++ post event activities (sharing pix, who which where; thanking host,...; check when is next available day and take actions: get someone to be an 'owner') .. check this card https://trello.com/c/zozx7OzE
+
+
+# Planning an event
+Creating a codebar event consists of seven steps which should follow this basic timeline (*details below*)
+1. Decide which organizer is responsible for the event.
+2. Find a host for the event.
+3. Create a card on Trello.
+4. Create event on codebar.io, as soon as the host selects a date.
+5. Post event on Meetup.com.
+6. Communicate with host
+7. Complete post-event activities.
+
+
+### 1. Sign up as event owner
+To ensure everyone an equal opportunity to participate at every level of the creation of events it is key that everyone can 'take the lead' of an event. One way of fostering such opportunity is to have each event have an 'owner'. By identifying a single person as the event owner, everyone, regardless of previous experience, temperament, tendencies or personality, etc, will have an equal opportunity to participate.
+
+This does not mean that the 'owner' has to do everything, though they can. Instead they have to check that each activity has been done and if necessary, bring our attention to the fact that things have not yet been completed. This will allow each and every organizer to participate in the process of creating events and communicating with hosts, not simple those who are accustomed to taking the lead.
+
+
+### 2. Find a host
++ TODO
+
+
+### 3. Create a card in Trello
+Procedures:
+
+Copy [this card](https://trello.com/c/6xVLVl5A) to set up a new workshop:
+
++ Adjust title to the correct date. Example: "Event 2018.03.08"
++ Set the due date on the card matching the event date
++ Make sure the "Event" blue tag is present
++ Remove the extra logos. Make the correct one the card cover.
++ Define the door responsible and the MC. Assign them as members of the card so they get notification of any changes.
```

Comment:
"Define the door responsible and the MC. Assign them as members of the card so they get notification of any changes." -> I would rephrase to: "Assign the card to you so the rest of the organisers can easily see you are the owner. Once defined, fill in the specific roles in the card description (MC, Door Person, and Pair responsible)"